content/integrations/cloud_foundry.md

```diff
@@ -197,7 +197,19 @@ addons:
 
 To see which `datadog-agent` release version you uploaded earlier, run `bosh releases`.
 
-If you don't have a local copy of the runtime configuration, create a `runtime.yml` with the current runtime configuration (`bosh runtime-config`) and add the above YAML to it. If the Director's runtime configuration is empty, create a new `runtime.yml` containing only the above YAML.
+#### Load the runtime.yml
+
+Check if you have a previously configured runtime-config by running:
```

Comment:
Is "runtime-config" a command? If so, `backticks`. Is it two words? If so, remove the hyphen.